aptos-move/framework/src/aptos-natives.bpl

```diff
@@ -77,3 +77,8 @@ axiom (forall limit: int :: {$1_aggregator_factory_spec_new_aggregator(limit)}
 axiom (forall limit: int :: {$1_aggregator_factory_spec_new_aggregator(limit)}
      (var agg := $1_aggregator_factory_spec_new_aggregator(limit);
      $1_aggregator_spec_aggregator_get_val(agg) == 0));
+
+// ==================================================================================
```

Comment:
@wrwg, since `function_info` belongs to aptos-framework, the uninterpreted procedure is added to `aptos-natives.bpl` instead of `prelude.bpl`. For the same reason, instead of adding a unit test to `third_party`, I tested it manually using CLI. cc @junkil-park